airbyte-integrations/connectors/source-jira/source_jira/streams.py

```diff
@@ -303,7 +303,6 @@ class Issues(IncrementalJiraStream):
 
     cursor_field = "updated"
     extract_field = "issues"
-    use_cache = True
```

Comment:
Can you please change 
```
use_cache = False
```
and add some comment that it was disable because of "reason"
?

Reply:
upated